examples/jupyter_lite_config.json

```diff
@@ -2,7 +2,7 @@
   "LiteBuildConfig": {
     "contents": ["."],
     "federated_extensions": [
-      "https://conda.anaconda.org/conda-forge/label/jupyterlite_pyodide_kernel_alpha/noarch/jupyterlite-pyodide-kernel-0.6.0a4-pyhd8cb729_0.conda",
+      "https://jupyterlite-pyodide-kernel--178.org.readthedocs.build/en/178/_static/jupyterlite_pyodide_kernel-0.6.0a4-py3-none-any.whl",
```

Comment:
I guess we can live it like this for now. Since this PR requires a change in `jupyterlite-pyodide-kernel` there will be a small window between the next JupyterLite 0.6.0 alpha and `jupyterlite-pyodide-kernel` where the file system access may not work over the service worker.

But once this is released we can update https://github.com/jupyterlite/pyodide-kernel/pull/178, merge and release it right after.